README.md

```diff
@@ -22,7 +22,7 @@ No more plaintext credentials in your home directory. Automatically authenticate
 
 ## 🪄 Usage
 
-![Example of 1Password Shell Plugins with AWS: user runs an `aws` command, a Touch ID prompt shows up, and `aws` is automatically authenticated](https://user-images.githubusercontent.com/7430639/205337855-41604aca-0ddb-4eab-a5f0-fb9107e09d8d.gif)
+![Example of 1Password Shell Plugins with AWS: user runs an `aws` command, a Touch ID prompt shows up, and `aws` is automatically authenticated](https://user-images.githubusercontent.com/1965218/206192140-cfa3103e-701c-46c1-9d7c-bd6a404b8823.gif)
```

Comment:
The `<picture>` element allows us to have older browser support.

```suggestion
<picture>
  <source srcset="https://user-images.githubusercontent.com/1965218/206192140-cfa3103e-701c-46c1-9d7c-bd6a404b8823.gif" type="image/webp" />
  <img src="https://user-images.githubusercontent.com/7430639/205337855-41604aca-0ddb-4eab-a5f0-fb9107e09d8d.gif" alt="Example of 1Password Shell Plugins with AWS: user runs an `aws` command, a Touch ID prompt shows up, and `aws` is automatically authenticated" />
<picture>
```

Reply:
Ah, thank you! I missed the `type` attribute.